infra/modules/providers/azure/app-monitoring/README.md

```diff
@@ -0,0 +1,117 @@
+
+The `action group` variables are only configurable for sending e-mail notifications when `metric alert criteria` crosses a configured threshold value. The values used to define `metric alert criteria` are provided by a template. If a template does not provide an e-mail by satisfying the `action_group_email_receiver` variable, all other configuration values will be ignored and this module will not be present in the Terraform plan generated for deployment:
+
+```terraform
+variable "resource_group_name" {
+  value = "test-rg"
+}
+
+variable "service_plan_name" {
+  value = "test-svcplan"
+}
+
+module "service_plan" {
+  resource_group_name     = "${var.resource_group_name}"
+  resource_group_location = "${var.resource_group_location}"
+  service_plan_name       = "${var.service_plan_name}"
+}
+
+module "app_monitoring" {
+  source                            = "../../modules/providers/azure/app-monitoring"
+  resource_group_name               = "${module.service_plan.resource_group_name}"
+  resource_ids                      = ["${module.service_plan.app_service_plan_id}"]
+  action_group_name                 = "${var.action_group_name}"
+  action_group_email_receiver       = "${var.action_group_email_receiver}"
+  metric_alert_criteria_namespace   = "${var.metric_alert_criteria_namespace}"
+  metric_alert_name                 = "${var.metric_alert_name}"
+  metric_alert_criteria_name        = "${var.metric_alert_criteria_name}"
+  metric_alert_criteria_aggregation = "${var.metric_alert_criteria_aggregation}"
+  metric_alert_criteria_operator    = "${var.metric_alert_criteria_operator}"
+  metric_alert_criteria_threshold   = "${var.metric_alert_criteria_threshold}"
+  scaling_values                    = "${var.scaling_values}"
+}
+```
+
+#### Configuring `metric alert criteria`
+
+Please visit _**Azure Monitor's**_ [Supported Metrics](https://docs.microsoft.com/en-us/azure/azure-monitor/platform/metrics-supported) page for a complete list of supported metrics per "Azure Resource".
+
+1. Choose an "Azure Resource" to be monitored.
+2. View the list of 'metrics' offered by the namespace of that "Azure Resource".
+3. Choose a metric.
+4. Define the metric's criteria (i.e. Conditional logic being applied to an "Azure Resource".)
+5. Apply that criteria to your chosen or newly created template.
+
+#### Configuring `action group`
+
+Please visit _**Azure Monitor's**_ [action-groups](https://docs.microsoft.com/en-us/azure/azure-monitor/platform/action-groups) page for more information on e-mail action groups.
+
+1. Choose an appropriate e-mail address to receive alert notifications.
+2. Choose an appropriate name for the 'action group' that will hold the e-mail address. (ex. "E-mail alert group")
+3. Apply the e-mail address and name using your chosen or newly created template.
+
+### Module Output
+
+Once the deployments are completed successfully, the output for the current module will be in the format mentioned below:
+
+```terraform
+Outputs:
```

Comment:
This is how I documented the exported attributes for another module as a reference https://github.com/microsoft/cobalt/tree/erisch/templates/azure-simple/feature/azkeyvault/infra/modules/providers/azure/traffic-manager#attributes-reference

Reply:
I like your way of documenting it because it provides a short explanation of what the output actually is. I'll make the same change in this Readme.